src/ess/sans/sans.py

```diff
@@ -0,0 +1,107 @@
+# SANS specific functions
+import scipp as sc
+import scippneutron as scn
+from .contrib import map_to_bins, to_bin_centers
+from .reduction import reduce_to_q, simple_reducer, grouping_reducer
+from .normalization import solid_angle, transmission_fraction
+
+
+def to_wavelength(
+    data,
+    transmission,
+    direct_beam,
+    direct,
+    masks,
+    wavelength_bins,
+    tof_bins_monitors,
+    pixel_size,
+    pixel_length,
+):
+    """
+    TOF to wavelength conversion
+    """
+    data = data.copy()
+    min_bin_mon2, max_bin_mon2, min_bin_mon4, max_bin_mon4 = tof_bins_monitors
+    transmission_frac = transmission_fraction(
+        transmission,
+        direct,
+        wavelength_bins,
+        min_bin_mon2,
+        max_bin_mon2,
+        min_bin_mon4,
+        max_bin_mon4,
+    )
+    for name, mask in masks.items():
+        data.masks[name] = mask
+    data = scn.convert(data, "tof", "wavelength", out=data, scatter=True)
```

Comment:
```suggestion
    data = scn.convert(data, "tof", "wavelength", scatter=True)
```
Recent versions of scippneutron do not support the `out` argument any more